common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -17,8 +17,7 @@
     % endif
     </div>
 
-    <fieldset role="${input_type}group" aria-label="${label}">
-
+    <fieldset role="${input_type}group" aria-label="${label.replace('"', '\'')}">
```

Comment:
@Shrhawk This Works for me.  Ideally, we would be using a ```<legend class="sr">``` here instead of the aria-label, but this should work fine.